lib/launcher.ts

```diff
@@ -251,7 +254,7 @@ let initFn = function(configFile: string, additionalConfig: Config) {
         let createNextTaskRunner = () => {
           let task = scheduler.nextTask();
           if (task) {
-            let taskRunner = new TaskRunner(configFile, additionalConfig, task, forkProcess);
```

Comment:
Totally! I'd be all for either `argv` or `commandLineFlags`|`commandLineConfig`.